src/services/createCurrentRoute.ts

```diff
@@ -18,7 +18,7 @@ export function createCurrentRoute(fallbackRoute: ResolvedRoute, push: RouterPus
     Object.assign(route, { ...newRoute })
   }
 
-  const currentRoute = readonly(route)
+  const currentRoute = route
```

Comment:
we _could_ rename `route` on line 15 to `currentRoute` and skip this const altogether?

Reply:
I was to revisit the readonly functionality of this. So let's just leave it for now.